src/block-components/alignment/edit.js

```diff
@@ -346,6 +348,18 @@ export const Edit = props => {
 	)
 }
 
+// Add additional classes when browser is Firefox to fix alignment
```

Comment:
You can move this code into `src/block-components/alignment/editor-has-polyfill.js` and then in `edit.js` you can just do an

```js
import './editor-has-polyfill'
```